torch/csrc/jit/passes/onnx/constant_fold.h

```diff
@@ -11,6 +11,7 @@ const int ONNX_OPSET_10 = 10;
 const int ONNX_OPSET_11 = 11;
 const int ONNX_OPSET_12 = 12;
 const int ONNX_OPSET_13 = 13;
+const int ONNX_OPSET_14 = 14;
```

Comment:
Do we use this value somewhere? I guess we don't nee them any more since we changed the if statement in constant_fold.cpp file.

Reply:
Although we might not use it currently, I think it might be a good idea to leave it in there for future use